lib/lint.ml

```diff
@@ -0,0 +1,310 @@
+module D = Dir_helpers
+
+module Checks = struct
+  type error =
+    | UnnecessaryField of string
+    | UnmatchedName of OpamPackage.Name.t
+    | UnmatchedVersion of OpamPackage.Version.t
+    | DubiousDuneSubst
+    | DuneProjectMissing
+    | DuneConstraintMissing
+    | DuneIsBuild
+    | BadDuneConstraint of string * string
+    | FailedToDownload of string
+    | WeakChecksum of string
+    | PinDepends
+    | ExtraFiles
```

Comment:
Seems like this is missing the prefix checks from https://github.com/ocurrent/opam-repo-ci/blob/b7fb27b10db7053b85aa2d3531fc86fa1516782e/lib/lint.ml#L44-L45  That may be intentional at this stage, but just making sure.

Reply:
Yes, that's intentional at this stage. There's a couple more (?) checks missing too, which I plan to add soon.